src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/entities/ResourceLinkEntity.kt

```diff
@@ -4,5 +4,6 @@ data class ResourceLinkEntity(
         var id: Int,
         var resourceContentFk: Int,
         var contentFk: Int?,
-        var collectionFk: Int?
+        var collectionFk: Int?,
+        var rc_fk: Int
```

Comment:
In the entities package, convention seems to be camelCase, as opposed to underscore naming in the DB.

Reply:
> Does this require changes in the [other repo](https://github.com/WycliffeAssociates/otter-common)? I don't see a PR there yet.

It did, I missed something. It is there now. [#83](https://github.com/WycliffeAssociates/otter-common/pull/83). The Content data model object already had the "end" field, and we don't yet have a Resource data model object, so there was only one file that needed changing and I didn't catch it.